tests/unit/runtimes/node/utils/node_version.js

```diff
@@ -7,11 +7,12 @@ const {
 } = require('../../../../../dist/runtimes/node/utils/node_version.js')
 
 test('getNodeVersion', (t) => {
+  t.is(getNodeVersion('nodejs14.x'), 14)
   t.is(getNodeVersion('nodejs12.x'), 12)
   t.is(getNodeVersion('nodejs8.x'), 8)
   t.is(getNodeVersion('12.x'), 12)
   t.is(getNodeVersion('8.x'), 8)
-  t.is(getNodeVersion('node14'), DEFAULT_NODE_VERSION)
+  t.is(getNodeVersion('node16'), DEFAULT_NODE_VERSION)
```

Comment:
```suggestion
  t.is(getNodeVersion('16.x'), DEFAULT_NODE_VERSION)
```

Reply:
I think this unit test check that something like `"nodeVersion"` is invalid so it uses the default one. Not sure if we should change it